music21/metadata/__init__.py

```diff
@@ -1224,34 +1154,51 @@ def search(self, query=None, field=None, **kwargs):
             except AttributeError:
                 pass
             if not match:
-                for searchAttribute in self.searchAttributes:
+                for uniqueName, value in self.all(skipContributors=True):
+                    if not self._isStandardUniqueName(uniqueName):
+                        # custom metadata, don't search it
+                        continue
```

Comment:
will this be found earlier in the `self. _getPluralAttribute(field)` line?  We would like to be able to search customMetadata if possible.

Reply:
No, custom metadata is currently not searchable.  If you want to search custom metadata there's a bunch of work needed in the search algorithm, because in some (most? all?) cases it will only report the first matching piece of metadata.  So a custom piece of metadata that has "title" somewhere in its custom name will mess your expected results up badly.  I'm not sure I want to bite that off just now.  It would be a pretty big behavior change.

Reply:
Ah.  That can be done later, I suppose, since it's not a feature in the current system apparently (I thought it was, but just checked).  I'll put that as a followup PR.  Your code is so clear that I'm pretty sure I know how to do it pretty easily.